ext/standard/tests/dns_get_record.phpt

```diff
@@ -0,0 +1,201 @@
+--TEST--
+array dns_get_record ( string $hostname [, int $type = DNS_ANY [, array &$authns [, array &$addtl [, bool &$raw = false ]]]] );
+--CREDITS--
```

Comment:
hi @weltling !
how are you?

i've search about your suggestion but i dont know how i can do that, can you give a example?
thanks, have a nice day!